SwaprHQ/swapr-dapp
# [SWA-15][FEAT] - Add zkSync Era support

# ⚠️ Waiting for SDK updates to be released ⚠️ 

## Fixes: [SWA-15](https://linear.app/swaprdev/issue/SWA-15/add-zksyncera)

# Description

* Add zkSync Era Mainnet and Testnet initial support
* Trade through **1inch**
* Bridge through **Socket**

![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/23e27d39-e199-4350-9120-c8c659502fcf)

# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr landing page
4) Connect your wallet to zkSync Era Mainnet
5) For now I couldn't get bridge routes nor trade quotas, but I can see the tokens, select them, import custom tokens (like $WETH outside Coingecko's list), and get token balances :c
